Atomic/util.py

```diff
@@ -1096,3 +1099,18 @@ def kpod(cmd, storage=None, debug=None):
     if debug:
         write_out(" ".join(cmd))
     return check_output(_kpod, env=os.environ)
+
+
+def remove_skopeo_prefixes(image):
+    """
+    Remove prefixes used by skopeo but not expected in other image uses.
+
+    :param image: The full image string
+    :type image: str
+    :returns: The image string without skopeo prefixes
+    :rtype: str
+    """
+    for remove in ('http:', 'https:'):
```

Comment:
this version looks better than what we do in `_get_skopeo_args` (_syscontainers.py). Could you please add the `oci:` prefix as well to the list and replace the code in `_get_skopeo_args` with a call to this function?

Reply:
Sure!